cmd/benchmark/main.go

```diff
@@ -215,11 +216,53 @@ func commandRemote(c *cli.Context) error {
 		}
 	}()
 
+	// Fetch latest nonce and block height
+	id := client.Public()
+	resp, err := client.GetAccountNonce(hex.EncodeToString(id[:]))
+	if err != nil {
+		return err
+	}
+
+	nonce := resp.Nonce
+	block := resp.Block
+
+	// Poll block height changes
+	consensus, err := client.PollLoggerSink(nil, wctl.RouteWSConsensus)
```

Comment:
For block height and nonce, care to have this done and initialised inside wctl, and remove the nonce and block height param in SendTransaction?